<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStockEntryDialog</class>
 <widget class="QDialog" name="AddStockEntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add stock</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Quantity:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Price:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="priceSpinBox">
       <property name="accelerated">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="priceKindCheckBox">
       <property name="text">
        <string>Price per item</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Storage:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="TreeViewComboBox" name="storageComboBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Condition:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="RelationalComboBox" name="conditionComboBox"/>
     </item>
     <item row="5" column="1">
      <widget class="QPlainTextEdit" name="commentTextEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Comment:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="quantitySpinBox">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <double>9999999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeViewComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/treeviewcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>RelationalComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/relationalcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>quantitySpinBox</tabstop>
  <tabstop>priceSpinBox</tabstop>
  <tabstop>priceKindCheckBox</tabstop>
  <tabstop>storageComboBox</tabstop>
  <tabstop>conditionComboBox</tabstop>
  <tabstop>commentTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddStockEntryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddStockEntryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
